### PR TITLE
Add Airflow-specific modifications - image can be build on its own

### PR DESCRIPTION
**Pull request by potiuk**
It uses only officially released code.

Part of https://github.com/apache/airflow/issues/9401

### CODE REVIEW

File: .idea/docker-openldap.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (turbaszek) [inline .idea/docker-openldap.iml]: We probably don't want any `.idea` content in the repo, right?

**Review by turbaszek — CHANGES_REQUESTED** [review]

**Comment by turbaszek** [comment]
Would it be a good idea to split this into two commits? (if I correctly understand)
- base code from osixia/openldap:1.2.0 
- airflow specific changes

**Comment by potiuk** [comment]
Correct @turbaszek - I split it into two commits. The first brings in the "base-image", the second adds airflow modifications.

**Review by turbaszek — APPROVED** [review]
LGTM

**Pull request merged**